packages/gatsby/src/index.js

```diff
@@ -0,0 +1,3 @@
+import Link, { withPrefix, navigateTo } from './components/link'
+
+export { Link, withPrefix, navigateTo }
```

Comment:
I'm not sure this is the best way to handle the imports here...I think we want an explicit `browser` entry file, so that all the server side code isn't accidently (in the future) imported. 

I would make a `browser.js` or maybe `client.js` (w/d/t @KyleAMathews) and point to that file in package.json `browser:` main field ([more info](https://github.com/defunctzombie/package-browser-field-spec))

Reply:
cool, never realized that `browser` field existed in package.json :smile: Thanks for pointing out. WIll probably wait for @KyleAMathews' comment and make `browser.js` or `client.js`.